integration/tests/epochs/suite.go

```diff
@@ -586,3 +617,20 @@ func (s *Suite) assertNetworkHealthyAfterANChange(ctx context.Context, env templ
 	require.Contains(s.T(), proposedTable.(cadence.Array).Values, cadence.String(info.NodeID.String()), "expected node ID to be present in proposed table returned by new AN.")
 
 }
+
+// assertNetworkHealthyAfterVNChange after an verification node is removed or added to the network
+// this func can be used to perform sanity.
+// 1. Ensure sealing continues by comparing latest sealed block from the root snapshot to the current latest sealed block
+func (s *Suite) assertNetworkHealthyAfterVNChange(ctx context.Context, _ templates.Environment, rootSnapshot *inmem.Snapshot, _ *StakedNodeOperationInfo)  {
+	bootstrapHead, err := rootSnapshot.Head()
+	require.NoError(s.T(), err)
+
+	snapshot, err := s.client.GetLatestProtocolSnapshot(ctx)
```

Comment:
This is returning the [latest finalized state](https://github.com/onflow/flow-go/blob/master/engine/access/rpc/backend/backend.go#L229), so by checking the height of `Head` (the latest finalized block), we aren't guaranteeing anything about the latest sealed block.

We could use [GetLatestBlock](https://github.com/onflow/flow-go/blob/3ec98807fbfa081e5f8613b01d6ed32f30dc4c65/access/api.go#L18) instead, with `isSealed` set to true.

Reply:
https://github.com/onflow/flow-go/pull/1763/commits/d7f322b3587614443f114e7c6bdf40be867acbf5